### PR TITLE
Clive 201732120167 user documentation

### PR DESCRIPTION
**Pull request by hema-001**

### CODE REVIEW

**Comment by hema-001** [comment]
@spm2020spring please ignore _DevDoc.md_

I don't know what happened, but all the previous commits from the [Developer documentations](https://github.com/lanlab-org/OAPS-Jifry/pull/5) branch, got included here, I'm unable to explain it.

Thanks
-IBRAHIM

**Comment by spm2020spring** [comment]
> @spm2020spring please ignore _DevDoc.md_
> 
> I don't know what happened, but all the previous commits from the [Developer documentations](https://github.com/lanlab-org/OAPS-Jifry/pull/5) branch, got included here, I'm unable to explain it.
> 
> Thanks
> -IBRAHIM

@hema-001 

I have no idea too.  

Perhaps it is because the changes in the previous pull request have not been merged to the master branch, and you pushed the same branch as you did in [pull request 5](https://github.com/lanlab-org/OAPS-Jifry/pull/5), i.e., `Clive-201732120167-UserDocumentation`.


Hui

File: Documents/UserDoc.md [hunk 1]
```diff
@@ -1 +1,126 @@
...
+**OAPS-Jifry** is a re-implementation of the **OAPS**(Open Access Publishing System)  **[TreeLib](https://github.com/youmowang/zjnuservice)**, using JSP technology. The original TreeLib was implemented using Python programming language.
+### Do i need to sign-up ?
+Absolutely NO.
+OAPS-Jifry, just like OAPS-TreeLib is open access system, that is, you don't need to sign-up, nor login in order to post/view articles. However, in OAPS-Jifry you can register as an author by the same email you used to post an article so you can edit, or delete your articles. This is one of the features that has been newly added to this version.
+### Why two implementations of the same system ?
+OAPS-TreeLib originated as a **OOAD** (Object Oriented Analysis and Design) fall 2018 course project, developed by Yun Dabang, Ni Zhongjie. Then, in fall 2019, the course project of OOAD was to maintain the same project of the previous year. Due to programming language barrier — the project was written by Python and most of 2019 students didn't learn or study Python. Jifry decided to re-implement the system from scratch using what he knows best **JSP**.
+### I have a question who to ask ?
```
Comment (spm2020spring) [inline Documents/UserDoc.md]: @hema-001 

Use "Who should I ask if I have questions?"

Hui

File: Documents/UserDoc.md [hunk 2]
```diff
@@ -1 +1,126 @@
...
+
+Specify the folder where you want Eclipse to be installed. The default folder will be in your User directory.
+
+Select the ‘Install’ button to begin the installation.
+
+### Launch Eclipse
+
+Once the installation is complete you can now launch Eclipse. The Eclipse Installer has done it's work. Happy coding.
+
+### Importing file
```
Comment (spm2020spring) [inline Documents/UserDoc.md:29]: @hema-001 

Not clear what files to import after reading this section.

Hui

File: Documents/UserDoc.md [hunk 3]
```diff
@@ -1 +1,126 @@
...
+
+Step 2:  **Inside the Java folder you will find a jdk-XX** folder, open it and open the  **bin** folder,  **copy the path of bin folder**.
+
+Note:- Now you have to set the environment variable.
+
+Step 3:  For that Go to **Control Panel > System and Security > System > Advanced System Setting > Environment Variables**
+
+Step 4:Under **System variable**, you will find “**PATH**” just select it and click on **edit.** Now click on **New** and **add the bin folder location**, after that click on “**OK**“.
+
+For the JSTL it is the same process
```
Comment (spm2020spring) [inline Documents/UserDoc.md]: @hema-001 

For the JSTL it is the same process.

Add a period in the end of the sentence.

Hui

File: Documents/UserDoc.md [hunk 4]
```diff
@@ -1 +1,126 @@
...
+Ensure the selected **JRE** is a full JDK and is of a version that will satisfy Apache Tomcat 1.  If necessary, you can click on  **Installed JREs...**  to add JDKs to Eclipse.
+Click  **Finish**  .
+
+### JDK  Jstl On windows 10
+Download from [oracle.com](https://www.oracle.com/technetwork/java/javase/downloads/index.html)
+Step 1:**  Open  **C Drive > Program Files**  folder and look for  **Java** folder and open it.
+
+Step 2:  **Inside the Java folder you will find a jdk-XX** folder, open it and open the  **bin** folder,  **copy the path of bin folder**.
+
+Note:- Now you have to set the environment variable.
```
Comment (spm2020spring) [inline Documents/UserDoc.md]: @hema-001 

Note: you have to add the above path to the environment variable PATH.

Hui

File: Documents/UserDoc.md [hunk 5]
```diff
@@ -1 +1,126 @@
...
+
+For the JSTL it is the same process
+
+### mysql-connector-java
+
+In eclipse, open **Window -> Show View -> Data Source Explorer**
+Right click on the above database option and select “New” for creating the new connection from the database
+Select the MySQL version to be configured
+If you don’t have the JAR files added, please add it manually by clicking on the “Add Jar/Zip” button.
+click on the button “Test Connection”, it will test the connection with database.
```
Comment (spm2020spring) [inline Documents/UserDoc.md]: @hema-001 

Click.  The first letter of a sentence should be in upper case.

Hui

File: Documents/UserDoc.md [hunk 6]
```diff
@@ -1 +1,126 @@
-# UserDoc
+# OAPS-Jifry User documentation
+
+
+## How to install and Setup the environment
```
Comment (spm2020spring) [inline Documents/UserDoc.md:4]: @hema-001 

How to set up environment to run OAPS-Jifry

Hui

File: Documents/UserDoc.md [hunk 7]
```diff
@@ -1 +1,126 @@
...
+Type your **email address**, and click "**Post Article**":
+![screenshot2](Documents/screenshots/s2.png)
+
+Fill in the following required fields:
+![screenshot3](Documents/screenshots/s3.png)
+Note:- The file format must be PDF only.
+
+Then, click "**Post**".
+
+You article now is successfully posted, and you can also see the article's description by clicking on the article's name from the subject list:
```
Comment (spm2020spring) [inline Documents/UserDoc.md]: @hema-001 

article's title.

Hui

File: Documents/UserDoc.md [hunk 8]
```diff
@@ -1 +1,126 @@
...
+
+Finally, the highlight is updated:
+![screenshot10](Documents/screenshots/s10.png)
+
+Of course, by clicking "**Delete**", the article will be deleted from the website.
+
+## FAQ
+In this frequently asked questions section, we attempt to answer some of the special questions that you may want to ask after reading this document.
+### What is OAPS-Jifry ?
+**OAPS-Jifry** is a re-implementation of the **OAPS**(Open Access Publishing System)  **[TreeLib](https://github.com/youmowang/zjnuservice)**, using JSP technology. The original TreeLib was implemented using Python programming language.
```
Comment (spm2020spring) [inline Documents/UserDoc.md:119]: @hema-001 

Is JSP a technology?  Not a programming language?

Hui

File: Documents/UserDoc.md [hunk 9]
```diff
@@ -1 +1,126 @@
...
+### Do i need to sign-up ?
```
Comment (spm2020spring) [inline Documents/UserDoc.md]: @hema-001 

Do I need to sign up to post an article?

Hui

File: Documents/UserDoc.md [hunk 10]
```diff
@@ -1 +1,126 @@
...
+Absolutely NO.
+OAPS-Jifry, just like OAPS-TreeLib is open access system, that is, you don't need to sign-up, nor login in order to post/view articles. However, in OAPS-Jifry you can register as an author by the same email you used to post an article so you can edit, or delete your articles. This is one of the features that has been newly added to this version.
```
Comment (spm2020spring) [inline Documents/UserDoc.md:122]: @hema-001 

What is the rationale of this added feature?

Hui

**Comment by hema-001** [comment]
> > @spm2020spring please ignore _DevDoc.md_
> > I don't know what happened, but all the previous commits from the [Developer documentations](https://github.com/lanlab-org/OAPS-Jifry/pull/5) branch, got included here, I'm unable to explain it.
> > Thanks
> > -IBRAHIM
> 
> @hema-001
> 
> I have no idea too.
> 
> Perhaps it is because the changes in the previous pull request have not been merged to the master branch, and you pushed the same branch as you did in [pull request 5](https://github.com/lanlab-org/OAPS-Jifry/pull/5), i.e., `Clive-201732120167-UserDocumentation`.
> 
> Hui

I don't think that is the reason. Because this is not the first time I create new pull request while there is another open pull request.

But, I think it's maybe because I did push two commits to pull request 5 before I create this new branch in the same command line session.

-IBRAHIM

File: Documents/UserDoc.md [hunk 11]
```diff
@@ -1 +1,126 @@
...
+### Why two implementations of the same system ?
+OAPS-TreeLib originated as a **OOAD** (Object Oriented Analysis and Design) fall 2018 course project, developed by Yun Dabang, Ni Zhongjie. Then, in fall 2019, the course project of OOAD was to maintain the same project of the previous year. Due to programming language barrier — the project was written by Python and most of 2019 students didn't learn or study Python. Jifry decided to re-implement the system from scratch using what he knows best **JSP**.
```
Comment (spm2020spring) [inline Documents/UserDoc.md]: @hema-001 

... originated as a course project for [Introduction to Object-Oriented Analysis and Design (Fall 2018)](http://lanlab.org/course/2018f/oo/project_description.pdf), developed by[ Yun Dabang and Ni Zhongjie](http://118.25.96.118/gsrycloud/TreeLib/).

Hui

**Comment by hema-001** [comment]
test

**Review by spm2020spring — APPROVED** [review]

**Pull request merged**